cssjson.js

```diff
@@ -222,3 +222,5 @@ var CSSJSON = new function () {
     };
 
 };
+
+module.exports = CSSJSON;

```

Comment:
This probably needs something like https://github.com/umdjs/umd since it can be used outside NodeJS.